frontend/src/features/homes/services/homes-list.js

```diff
@@ -0,0 +1,72 @@
+import { $axios } from "src/boot/axios";
+import { Notify } from "quasar";
+import { errorNotifier } from "src/utils/notifier.js";
+import { i18n } from "src/boot/i18n";
+
+export const getUserGroups = async () => {
+  try {
+    const { data } = await $axios.post("/methods/currentUserGroups");
+    return data;
+  } catch (error) {
+    errorNotifier(error);
+    return [];
+  }
+};
+
+export const getHomesWithActivityLevel = async groupId => {
+  try {
+    const { data } = await $axios.post("/methods/getHomesWithActivityLevel", {
+      groupId
+    });
+    return data;
+  } catch (error) {
+    errorNotifier(error);
```

Comment:
Perhaps this `errorNotifier` function could be responsible for creating the UI notifications?

Reply:
Yes it is a UI notifier only